In [2]:
#import the libraries
import pandas as pd
import scipy 
import numpy as np
from scipy import stats

In [5]:
dataframe df = pd.read_csv('/content/Cutlets.csv')

In [7]:
df.head(10)

Unit A  Unit B
0  6.8090  6.7703
1  6.4376  7.5093
2  6.9157  6.7300
3  7.3012  6.7878
4  7.4488  7.1522
5  7.3871  6.8110
6  6.8755  7.2212
7  7.0621  6.6606
8  6.6840  7.2402
9  6.8236  7.0503

In [21]:
unitA = pd.Series(df.iloc[:,0])
unitA.head(5)

0    6.8090
1    6.4376
2    6.9157
3    7.3012
4    7.4488
Name: Unit A, dtype: float64

In [23]:
unitB = pd.Series(df.iloc[:,1])
unitB.head(5)

0    6.7703
1    7.5093
2    6.7300
3    6.7878
4    7.1522
Name: Unit B, dtype: float64

In [32]:
p=scipy.stats.ttest_ind(unitA, unitB)

In [33]:
p

Ttest_indResult(statistic=0.7228688704678061, pvalue=0.4722394724599501)

In [36]:
p_value = p[1]
p_value

0.4722394724599501

Since P_value generated 0.47 > 0.05 hence there is no difference between the diameter of the two unit's cutlets